library/core/src/mem/maybe_uninit.rs

```diff
@@ -573,6 +574,28 @@ impl<T> MaybeUninit<T> {
         }
     }
 
+    /// Drops the contained value in place.
+    ///
+    /// If you have ownership of the `MaybeUninit`, it is preferable to use
+    /// [`assume_init`] instead, which prevents duplicating the content.
```

Comment:
Hm, this seems false, as drop_in_place is more efficient than moving out of the value and then dropping it, in general.

Reply:
Ah oops, I copied that from the `read` function. Replaced it with the same note that's on `ManuallyDrop::drop`.